src/sentry/utils/distutils/commands/build_integration_docs.py

```diff
@@ -19,4 +19,4 @@ def get_dist_paths(self):
     def _build(self):
         from sentry.utils.integrationdocs import sync_docs
         log.info('downloading integration docs')
-        sync_docs()
+        sync_docs(nice=True)
```

Comment:
The naming `nice` is a bit awkward. I also think it's not even worth making this an argument since it's nothing else hooks into this.

I think if we want to allow this to fail, we just let it fail always.